types/spectest/tests/consensusdata/deneb_blinded_block_validation.go

```diff
@@ -0,0 +1,6 @@
+package consensusdata
+
+// DenebBlindedBlockValidation tests a valid consensus data with deneb blinded block
+func DenebBlindedBlockValidation() *SpecTest {
```

Comment:
do we want to merge without all these tests?

Reply:
`ConsensusData` has no implemented tests yet, only for versioned blocks inside the `proposer` folder, which were altered to use Deneb instead of Capella.
However, there's the PR https://github.com/bloxapp/ssv-spec/pull/344 for that (`ConsensusData` tests).
We can also merge it before the spec release.
What do you think?

Reply:
I approved #334 so we can merge it , but then we'll have to impl these tests?  we want to get this merged and release asap

Reply:
Yes, after merging, we'll need to implement the missing tests for Deneb but it shall be fast to do

Reply:
done